caldav/server.go

```diff
@@ -36,6 +36,8 @@ type Backend interface {
 	GetCalendar(ctx context.Context, path string) (*Calendar, error)
 
 	GetCalendarObject(ctx context.Context, path string, req *CalendarCompRequest) (*CalendarObject, error)
```

Comment:
Can we remove `GetCalendarObject`, to avoid having two functions which do a very similar thing? `GetCalendarObject` is equivalent to `GetCalendarObjects` with a single path.

Reply:
PR updated: Replaced GetAddressObject & GetCalendarObject with GetAddressObjects & GetCalendarObjects and its demand in Backend interface too.